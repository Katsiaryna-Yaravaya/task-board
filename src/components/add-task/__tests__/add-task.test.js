import React from 'react'
import { AddTask } from '../../index'
import { shallow } from 'enzyme'
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import { Provider } from 'react-redux'
import configureStore from 'redux-mock-store'
import thunk from 'redux-thunk'

Enzyme.configure({ adapter: new Adapter() })

const middlewares = [thunk]
const mockStore = configureStore(middlewares)

describe('add-task', () => {
  it('add-task shallow', () => {
    const tree = shallow(
      <Provider store={mockStore()}>
        <AddTask />
      </Provider>
    )
    expect(tree).toMatchSnapshot()
  })
})
