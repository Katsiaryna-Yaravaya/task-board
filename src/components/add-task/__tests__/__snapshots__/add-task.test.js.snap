// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add-task add-task shallow 1`] = `ShallowWrapper {}`;
